packages/core/src/Provider/index.ts

```diff
@@ -9,4 +9,3 @@ export * from './UtxoProvider';
 export * from './ChainHistoryProvider';
 export * from './providerFactory';
 export * from './types';
-export * from './HandleProvider';
```

Comment:
Interface should remain in `core`, only the implemenation should be moved to `wallet`